features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -83,13 +83,13 @@
 #ifdef LWIP_DEBUG
 #define TCPIP_THREAD_STACKSIZE      1200*2
 #else
-#define TCPIP_THREAD_STACKSIZE      1200
```

Comment:
Hi @bulislaw I tried to add a macro control to lwipopts, and I also put our target specific definitions in our lwipopts_conf.h.